articles/how-crypto-mining-works/index.md

```diff
@@ -0,0 +1,75 @@
+### How Does Cryptocurrency Mining Work?  
+It is hard to talk about cryptocurrency without mentioning mining. Mining cryptocurrency is a complex mathematical system, generally utilizing cryptographic hashing algorithms. For the purposes of this article, Bitcoin mining at a high level will be examined. 
+
+### What Does Mining Do for the Cryptocurrency?  
+Mining can be thought of as the "processor" for transactions on the blockchain. [Bitcoin.org](bitcoin.org/en/how-it-works) describes the process of mining as "a distributed consensus system that is used to confirm pending transactions by including them in the block chain". Transactions are packaged into a block and then mined. This process of "mining" includes the transactions and verifies them in the chain. By continuing mining, these transactions are confirmed and become nearly unmodifiable by attacks on the chain. 
+
+In Bitcoin, every block is connected, as the previous block hash is used to generate a new block. If a hacker wanted to modify or "double-spend"
+
+### What Does A User Get Out of Mining?  
+While the chain and the cryptocurrency itself, benefits from users mining, users also benefit. A "miner" receives a reward for their work. This reward is generally a step amount per block and the fees of transactions that are included in the block. 
+
+The distributed mining power of the network also protects the blockchain from attacks, like double spending or any other transaction modifying attacks. In order for these attacks to be successful, the attacker would have to rehash the block they want to modify and make the longest valid chain of blocks (highest combined difficulty of the chain of blocks). This not only becomes harder the more blocks added after the attack block, but would likely require 51% of the total hashing power.
+
+### What is Proof-of-Work?  
+Bitcoin uses a proof-of-work algorithm. These algorithms provide a computational proof that can be easily checked. Most often, proof of work is implementing using various hashing algorithms with a target value. In order to successfully mine a block, a miner must generate a hash that is less than the target value with a certain difficulty value.
```

Comment:
Define what the target value is. It is not clear what that means? Assume the reader has no prior knowledge. The last sentence says we are comparing two hashes. How can we compare 2 hashes? Aren't hashes alphanumeric strings? What is a difficulty value?

Reply:
hashes are hexademical, thus yes they are alphanumeric 0-9, a-f